cscs-checks/prgenv/helloworld.py

```diff
@@ -16,12 +16,22 @@ class HelloWorldBaseTest(rfm.RegressionTest):
     prgenv_flags = {}
     sourcepath = 'hello_world'
     build_system = 'SingleSource'
+    prebuild_cmds = ['RFM_BUILD_TIME="$(date +%s%N)"']
+    postbuild_cmds = [
+        'RFM_BUILD_TIME="$(($(date +%s%N)-RFM_BUILD_TIME))"',
```

Comment:
Just use `date +%s` since you are going to use only the seconds.

Reply:
I think (I am rerunning to check) some compilations could lasts less than a second, so I think it's better to have a float than `0` compilation time. What do you think?

Reply:
Then it's fine. Just remove the unused imports and it's ready